aboutMe/apps.js

```diff
@@ -0,0 +1,55 @@
+//Q1
+const page=document.querySelector("page");
+style.fontfamily="Arial, sans-serif"
+//Q2
+const nickname = document.querySelector('#nickname');  
+const favorite = document.querySelector('#favorite'); 
+const hometown = document.querySelector('#hometown'); 
+
+nickname.innerHTML = "rayG";
+favorite.innerHTML = "playing video games"; 
+hometown.innerHTML = "Riyadh";
+
+const li=document.querySelector('li');
+for(let listitem= 
+  listitem.class.add('listitem');
+   listitemstyle.color="rebeccapurple";
+}
+
+//q3,4
+const pic=document.createElement('img'); ('src', 'https://picsum.photos/id/214/200/300')
```

Comment:
You forget to append the pic to the div.
```suggestion
const pic=document.createElement('img'); 
pic.setAttribute('src', 'https://picsum.photos/id/214/200/300')
const profilepicture=document.querySelector('.profilePicture');

profilepicture.appendChild(pic)
```